app/views/expenses/RemoveExpensesView.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.employment.routes.EmploymentSummaryController
+@import controllers.expenses.routes.RemoveExpensesController
+@import views.html.templates.Layout
+@import views.html.templates.helpers.{Heading, Link}
+
+@this(
+    layout: Layout,
+    heading: Heading,
+    link: Link,
+    formWithCsrf: FormWithCSRF,
+    govUkButton: GovukButton
+)
+@(taxYear: Int)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+
+@titleForUse = @{
+    messages(s"expenses.removeExpenses.title.${if(user.isAgent) "agent" else "individual"}")
+}
+
+@headingForUse = @{titleForUse}
+
+@layout(pageTitle = titleForUse, taxYear = Some(taxYear), isAgent = user.isAgent) {
+    @formWithCsrf(action = controllers.expenses.routes.RemoveExpensesController.submit(taxYear)) {
+        @heading(headingForUse, Some(messages("expenses.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-3")
```

Comment:
the screenshots show the caption as "Employment for.." instead of "Employment expenses for..."